docs/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```

Comment:
Beware of the hottest 🔥 frontend framework today https://htmx.org/

Reply:
That's cool and actually good to know. :)